app/code/Magento/Inventory/Test/Integration/Indexer/IndexationTest.php

```diff
@@ -50,17 +51,25 @@ class IndexationTest extends TestCase
     private $indexer;
 
     /**
-     * @var Checker
+     * @var GetProductQuantityInStockInterface
      */
-    private $indexerChecker;
+    private $getProductQtyInStock;
 
+    /**
+     * {@inheritdoc}
+     */
     protected function setUp()
     {
         $this->indexer = Bootstrap::getObjectManager()->create(Indexer::class);
         $this->indexer->load(StockItemIndexerInterface::INDEXER_ID);
-        $this->indexerChecker = Bootstrap::getObjectManager()->create(Checker::class);
+        $this->getProductQtyInStock = Bootstrap::getObjectManager()->create(
```

Comment:
Pls replace on `get` method
We do not need to create new object
Our services are stateless and are shared objects
Thanks